.github/CODEOWNERS

```diff
@@ -30,7 +30,7 @@
 /sdk/identity/                                                       @mccoyp @schaabs @xiangyan99
 
 # PRLabel: %Event Hubs
-/sdk/eventhub/                                                       @annatisch @yunhaoling @swathipil @rakshith91
+/sdk/eventhub/                                                       @annatisch @kashifkhan @swathipil @rakshith91
```

Comment:
Should we consider adding Libba to Event Hubs and Service Bus?

Reply:
Added.